src/styles/common/_fonts.scss

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+//Primary Font
+$dell-font: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+// Dell Design Standards Font Weights
+$font-weight-light: 300;
```

Comment:
Light should be 200 ?

Reply:
https://web.ddsproduct.com/31b3fd8b1/p/578bb9-typography/b/195c15
Followed this as per DDS Light 300, Regular 400, Medium 500